README.md

```diff
@@ -288,6 +288,30 @@ OpenMock leverages [https://golang.org/pkg/text/template/](https://golang.org/pk
   {{.HTTPBody | xmlPath "node1/node2/node3"}}
   ```
 
+# Admin Interface
+Openmock also opens an admin interface on port 9998, which allows you to see and 
+manipulate the loaded templates.
+
+## Get Templates
+`curl -X GET http://localhost:9998/api/v1/templates`
+Prints YAML representation of all the templates the instance has loaded
+
+## Post templates
+```
+curl -Xcurl -X POST \
+  http://localhost:9998/api/v1/templates \
+  -d '<template yaml>'
+```
+Add template(s) (per the posted YAML) to the instance
+
+### Post helper
+There is an additional executable provided by openmock to help post templates
+at a remote instance.  Use the command: 
+```
+OPENMOCK_TEMPLATES_DIR=./demo_templates OPENMOCK_ENDPOINT=http://localhost:9998 om_post_templates
```

Comment:
nice, does it concatenate all the yaml files?

Reply:
nvm, it uses `Load()`

Reply:
I'd love to aim for a better interface here, e.g. `omctl directory [server]`